### PR TITLE
Fix for Placeholder IE bug

### PR DESCRIPTION
**Pull request by neborn**
Currently only IE 9 has the desired cursor movement. But IE7 and 8 at least no longer have javascript errors.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,4 @@ before_script:
 script: grunt lint; grunt qunit
 notifications:
   email:
-  - kirollos@gmail.com
-  - jgourgoutis@linkedin.com
-  - iheartweb@gmail.com
-  - rockthemic12@gmail.com
```
Comment (Iheartweb) [inline .travis.yml:13]: uh-oh.

File: scripts/components/Placeholder.js [hunk 2]
```diff
@@ -60,16 +60,24 @@ Placeholder = FormElement.extend( function ( base ) {
...
         if ( Placeholder.hasState( PLACEHOLDER_STATE ) ) {
           // Prevent default prevents the placeholder text from being highlighted on tab focus
           evt.preventDefault();
-          if ( $element[0].createTextRange ) {
-            range = $element[0].createTextRange();
-            range.moveat( 'character', 0 );
-            range.movend( 'character', 0 );
-          } else if ( $element[0].setSelectionRange ) {
-            $element[0].setSelectionRange( 0, 0 );
+          element = $element[0];
```
Comment (Iheartweb) [inline scripts/components/Placeholder.js:67]: http://stackoverflow.com/questions/7745867/how-do-you-get-the-cursor-position-in-a-textarea

File: .travis.yml (hunk 1 above)
Comment (neborn) [inline .travis.yml:13]: I don't know what happened. I thought I was only making changes to my fork of the repo, and this commit somehow got added to my pull request (the commit occurred after my pull request). Sorry.

Comment (neborn) [inline .travis.yml:13]: And a question: if .travis.yml is in the .gitignore file, how come it's still being tracked?

Comment (Iheartweb) [inline .travis.yml:13]: hmm... I have no clue! And, no worries.

**Pull request closed without merging**